src/layer/riscv/bias_riscv.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2025 AtomAlpaca <atal@anche.no>. All rights reserved.
+//
+// Licensed under the BSD 3-Clause License (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// https://opensource.org/licenses/BSD-3-Clause
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#include "bias_riscv.h"
+
+#if __riscv_vector
+#include <riscv_vector.h>
+#endif // __riscv_vector
+
+#include "cpu.h"
+
+namespace ncnn {
+
+Bias_riscv::Bias_riscv()
+{
+#if __riscv_vector
+    support_packing = true;
+#endif // __riscv_vector
+#if NCNN_ZFH
+#if __riscv_vector
+    support_fp16_storage = cpu_support_riscv_zvfh();
+#else
+    support_fp16_storage = cpu_support_riscv_zfh();
+#endif
+#endif
+}
+
+#if __riscv_vector
+static inline vfloat32m8_t __riscv_vfadd_vf_f32m8_bias(vfloat32m8_t op1, float bias, size_t vl)
+{
+    return __riscv_vfadd_vf_f32m8(op1, bias, vl);
+}
+#endif // __riscv_vector
```

Comment:
vfadd does not require writing an additional independent function
abs is done because there is no direct vfabs intrinsic